qa/tasks/mgr/dashboard/test_cephfs.py

```diff
@@ -347,3 +348,21 @@ def test_statfs(self):
         self.assertEqual(stats['subdirs'], 1)
 
         self.rm_dir('/animal')
+
+    def test_cephfs_clients_get_after_mds_down(self):
```

Comment:
function name: test_cephfs_clients_get_after_mds_down -> test_up_mds_counter_after_fs_fail, is ok? @dparmar18